kernel/enter.c

```diff
@@ -203,6 +203,30 @@ static int _setup_ext2(const char* rootfs)
 }
 #endif /* MYST_ENABLE_EXT2FS */
 
+#if defined(MYST_ENABLE_HOSTFS) && defined(MYST_DEBUG)
+static int _setup_hostfs(const char* rootfs)
+{
+    int ret = 0;
+
+    if (myst_init_hostfs(&_fs) != 0)
+    {
+        myst_eprintf("failed initialize the RAM files system\n");
+        ERAISE(-EINVAL);
+    }
+
+    if (myst_mount(_fs, rootfs, "/") != 0)
+    {
+        myst_eprintf("cannot mount root file system\n");
+        ERAISE(-EINVAL);
+    }
+
+    _create_standard_directories();
```

Comment:
What's the impact of the created folders? Are they written back to the host?

Reply:
They are written back to the hostfs source directory. This will vanish after we have RAMFS file systems for /tmp and /proc.